controllers/common-controller-utils_test.go

```diff
@@ -148,45 +178,49 @@ func getRamenConfig(t *testing.T, ctx context.Context, fakeClient client.Client)
 	return ramenConfig
 }
 
-func getRamenS3Secret(secretName string, ctx context.Context, fakeClient client.Client) (corev1.Secret, error) {
+func getRamenS3Secret(secretName string, ctx context.Context, fakeClient client.Client, ramenNamespace string) (corev1.Secret, error) {
 	ramenSecret := corev1.Secret{}
 	namespacedName := types.NamespacedName{
 		Name:      common.CreateUniqueSecretName(secretName, StorageClusterNamespace, StorageClusterName, common.S3ProfilePrefix),
-		Namespace: common.RamenHubNamespace,
+		Namespace: ramenNamespace,
 	}
 	err := fakeClient.Get(ctx, namespacedName, &ramenSecret)
 
 	return ramenSecret, err
 
 }
 
-func expectedRamenConfig() rmn.RamenConfig {
-	return rmn.RamenConfig{
-		S3StoreProfiles: []rmn.S3StoreProfile{
-			{
-				S3ProfileName:        fmt.Sprintf("%s-%s-%s", common.S3ProfilePrefix, TestSourceManagedClusterName, StorageClusterName),
-				S3Bucket:             TestS3BucketName,
-				S3CompatibleEndpoint: TestS3RouteHost,
-				S3Region:             "",
-				S3SecretRef: corev1.SecretReference{
-					Name:      common.CreateUniqueSecretName(TestSourceManagedClusterName, StorageClusterNamespace, StorageClusterName, common.S3ProfilePrefix),
-					Namespace: common.RamenHubNamespace,
-				},
+func getS3Profile(ramenNamespace string, sourceManagedClusterName string, destinationManagedClusterName string) []rmn.S3StoreProfile {
+	return []rmn.S3StoreProfile{
+		{
+			S3ProfileName:        fmt.Sprintf("%s-%s-%s", common.S3ProfilePrefix, sourceManagedClusterName, StorageClusterName),
+			S3Bucket:             TestS3BucketName,
+			S3CompatibleEndpoint: TestS3RouteHost,
+			S3Region:             "",
+			S3SecretRef: corev1.SecretReference{
+				Name:      common.CreateUniqueSecretName(sourceManagedClusterName, StorageClusterNamespace, StorageClusterName, common.S3ProfilePrefix),
+				Namespace: ramenNamespace,
 			},
-			{
-				S3ProfileName:        fmt.Sprintf("%s-%s-%s", common.S3ProfilePrefix, TestDestinationManagedClusterName, StorageClusterName),
-				S3Bucket:             TestS3BucketName,
-				S3CompatibleEndpoint: TestS3RouteHost,
-				S3Region:             "",
-				S3SecretRef: corev1.SecretReference{
-					Name:      common.CreateUniqueSecretName(TestDestinationManagedClusterName, StorageClusterNamespace, StorageClusterName, common.S3ProfilePrefix),
-					Namespace: common.RamenHubNamespace,
-				},
+		},
+		{
+			S3ProfileName:        fmt.Sprintf("%s-%s-%s", common.S3ProfilePrefix, destinationManagedClusterName, StorageClusterName),
+			S3Bucket:             TestS3BucketName,
+			S3CompatibleEndpoint: TestS3RouteHost,
+			S3Region:             "",
+			S3SecretRef: corev1.SecretReference{
+				Name:      common.CreateUniqueSecretName(destinationManagedClusterName, StorageClusterNamespace, StorageClusterName, common.S3ProfilePrefix),
+				Namespace: ramenNamespace,
 			},
 		},
 	}
 }
 
+func expectedRamenConfig(ramenNamespace string, source string, destination string) rmn.RamenConfig {
```

Comment:
`ramenNamespace string, source string, destination string` to `ramenNamespace, source, destination string`

Reply:
ack